types/query-string/query-string-tests.ts

```diff
@@ -19,20 +19,26 @@ import * as queryString from 'query-string';
     result = queryString.stringify({ foo: 'bar' }, { strict: false, encode: false });
 }
 
+// For each section below, the second line ensures the real answer is of the declared
```

Comment:
You can also use `// $ExpectType`. https://github.com/Microsoft/dtslint#write-tests

Reply:
Ooh, didn't know about that. `$ExpectType` won't work for the tests below though, right?

Reply:
In all the below tests the output is of type `queryString.OutputParams`, so it looks like they're just testing that certain things are assignable to that? Then yeah, `// $ExpectType` wouldn't work well.

Reply:
Got it. So to confirm, a test could be written to be this?

```ts
    // queryString.parse('?foo=bar');
    // $ExpectType queryString.OutputParams
    {foo: "bar"};
```

Reply:
No, because `// $ExpectType` just matches the stringified type with the expected string and doesn't consider subtyping. (The type of `{foo: "bar"}` is `{ foo: string }`.) For considering assignability of an object literal to a named type what you're doing now is fine.

Reply:
Got it, thanks for clarifying!